src/transxchange/read.rs

```diff
@@ -177,6 +316,14 @@ fn read_xml(transxchange: &Element, collections: &mut Collections) -> Result<()>
     }
     let companies = load_companies(transxchange)?;
     collections.companies.merge(companies);
+    let (commercial_mode, physical_mode) = load_commercial_physical_modes(transxchange)?;
```

Comment:
Warn and continue if mode absent in xml or not in MODES list.

Reply:
I discussed with @papailio and possibly, we're gonna end up with a default mode when it's not present or incorrect. That said, the default value has not been defined yet. Therefore, I did refactor the code to make it easy to ignore completely one file at any point of the parsing (in this case, if `load_commercial_physical_modes()` returns an error.

Reply:
Done in 9837c714c2044c279627152b80c554924f5198d8.